### DISCUSSION TITLE
锟斤拷烫烫烫

### DISCUSSION THREAD

**Discussion by Error1015** [Meloong-Git/PCL#3155] [提问] [answered]
搞了一个百度网盘的破解文件 然后发现打开pcl报错
![image](https://github.com/Hex-Dragon/PCL2/assets/156535541/84d7fd9e-328d-4ee4-a60a-f9df7da9a5b7)

**Comment by Error1015** [comment]
就不该瞎搞的哎 window爆了个木马病毒 明显不是pcl的

**Reply by WorldHim** [reply]
报毒是正常现象，打不开可以尝试更新

**Comment by allMagicNB** [comment]
你PCL2不是最新的，请尝试更新

**Comment by MoYuan-CN** [comment, marked as answer]
请通过官方渠道下载 PCL：https://afdian.net/p/0164034c016c11ebafcb52540025c377

**Reply by MoYuan-CN** [reply]
如果通过此渠道下载的最新版 PCL 仍然有问题，请补充

**Reply by allMagicNB** [reply]
或下载GitHub上的[最新正式版.zip](https://github.com/Hex-Dragon/PCL2/blob/main/%E6%9C%80%E6%96%B0%E6%AD%A3%E5%BC%8F%E7%89%88.zip)

**Comment by WForst-Breeze** [comment]
应该是下载的破解软件有问题，杀软杀个毒之后再试试

提问者貌似意为下载了一个别的软件的破解版后扫描出木马并无法打开 PCL，而非下载了破解版 PCL 并对 PCL 报毒

**Reply by allMagicNB** [reply]
[火绒](https://huorong.cn/downloadv5.html?status=hrstat&src=19)

**Reply by Error1015** [reply]
莫名其妙弄好了 非常感谢你们的帮助!
